integration_tests/tsconfig.json

```diff
@@ -3,12 +3,16 @@
     "target": "es5",
     "noEmit": true,
     "lib": ["es5", "dom", "es2015.promise"],
-    "types": ["cypress", "approved-premises"],
+    "types": ["cypress"],
     "esModuleInterop": true,
     "typeRoots": ["../server/@types", "../node_modules/@types"],
     "resolveJsonModule": true,
-    "baseUrl": "./",
     "moduleResolution": "node",
+    "baseUrl": ".",
+    "paths": {
+      "@approved-premises-ui": ["../server/@types/ui/index.d.ts"],
+      "@approved-premises-shared": ["../server/@types/shared/index.d.ts"]
```

Comment:
Could we have `@approved-premises/ui` and `@approved-premises/api` or something?

Reply:
I deffo think there's scope for breaking this out into a module of its own at some point, especially as CAS3 will be forking at some point

Reply:
Will also mean that we can generate new types and schedule releases automatically. That way if there are any major changes to the API, we're alerted via Renovate